www-client/elinks/elinks-9999.ebuild

```diff
@@ -17,13 +17,14 @@ KEYWORDS=""
 IUSE="bittorrent brotli bzip2 debug finger ftp gopher gpm guile idn ipv6
 	javascript libressl lua +mouse nls nntp perl ruby samba ssl tre unicode X xml zlib"
 
+BDEPEND="virtual/pkgconfig"
 RDEPEND="
 	brotli? ( app-arch/brotli )
 	bzip2? ( >=app-arch/bzip2-1.0.2 )
 	gpm? ( >=sys-libs/ncurses-5.2:0= >=sys-libs/gpm-1.20.0-r5 )
 	guile? ( >=dev-scheme/guile-1.6.4-r1[deprecated] )
 	idn? ( net-dns/libidn )
-	javascript? ( >=dev-lang/spidermonkey-1.8.5:0= )
+	javascript? ( dev-lang/spidermonkey:17= )
```

Comment:
:17 looks really old to me. Is this going to be improved or go away any time soon?